src/vm/i386/excepx86.cpp

```diff
@@ -3731,4 +3752,10 @@ AdjustContextForVirtualStub(
     return TRUE;
 }
 
+#ifdef FEATURE_PAL
+VOID DECLSPEC_NORETURN DispatchManagedException(PAL_SEHException& ex, bool isHardwareException)
```

Comment:
Ok, I guess the dummy dispatcher will be easier to remove once we move to WIN64EXCEPTIONS in the future. But I would recommend not to implement is as throw, but rather put in a DebugBreak();. The throw cannot work in most of the cases since it would hit managed frames where the C++ exception handling would get lost. So it would be better to break into the debugger or crash right on the spot if  there is no debugger attached.